lib/tape/ansible_runner.rb

```diff
@@ -41,7 +41,11 @@ class AnsibleRunner < ExecutionModule
     "Checks out app code, installs dependencies and restarts unicorns for "\
     "both FE and BE code."
   action :everything,
-    proc { ansible if valid_preconfigs },
+    proc {
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Avoid using `{...}` for multi-line blocks.